content/en/security/application_security/enabling/serverless.md

```diff
@@ -168,6 +221,12 @@ To install and configure the Datadog Serverless Framework plugin:
      DD_EXPERIMENTAL_ENABLE_PROXY: true
      AWS_LAMBDA_EXEC_WRAPPER: /opt/datadog_wrapper
```

Comment:
Can we also add the following environment variable to the Node and Python section?

DD_TRACE_ENABLED:true